fluent-syntax/tests/fixtures/normalized/attributes.ftl

```diff
@@ -0,0 +1,17 @@
+attribute =
```

Comment:
Thought: Yeah, seeing .ftl files it's a more maintainable solution compared to the inline macros.